### PR TITLE
Fix #81 refresh upon pressing enter in editor font size box

### LINKED ISSUE #81: Hitting enter inside editor font box takes user to homepage
In fixing issue #25 I prevented the form containing the font box from submitting, however, it seems like something similar is happening again.

### PR DESCRIPTION
**Pull request by wchill**
Fixes #81. Verified on OS X 10.10, Chrome 41 and Safari 8.0.3.

### CODE REVIEW

**Comment by neelabhg** [comment]
Great! Thanks for taking care of this so quickly.

I suggest avoiding the use of `return false;`. See ["jQuery Events: Stop (Mis)Using Return False"](http://fuelyourcoding.com/jquery-events-stop-misusing-return-false/) and ["The difference between ‘return false;’ and ‘e.preventDefault();’"](https://css-tricks.com/return-false-and-prevent-default/).

While `return false;` gets the job done, using it makes it harder to understand the intention. So, as in second link, `return false;` can be replaced with
```
e.preventDefault();
e.stopPropagation();
```
This will make it obvious that we are trying to prevent default behavior as well as stop the event from bubbling.

**Comment by wchill** [comment]
That change appears to work on my local machine. Should be good now.

File: app/scripts/main.js [hunk 1]
```diff
@@ -189,7 +189,9 @@ $(document).ready(function () {
         });
 
     $('#editor-opts-container').find('form').submit(function (e) {
+        //return false;
```
Comment (neelabhg) [inline app/scripts/main.js]: Shouldn't this be removed?

Comment (wchill) [inline app/scripts/main.js]: Oops. I'll take care of it.

**Pull request merged**